### PR TITLE
add Merge Strategy section to README

### PR DESCRIPTION
**Pull request by vorburger**
@ptuomola @xurror @awasum @avikganguly01 @thesmallstar @percyashu @fynmanoj @vidakovic @conradsp @Grandolf49 how about (something like) this to document the recommended _Merge Strategy_ for this project? Your review feedback, if any, most welcome, of course.

### CODE REVIEW

**Review by awasum — APPROVED** [review]
LGTM. Just check the typo as the last line: `Multiple commits in singl PR make sense in certain cases (e.g. branch backports).`

**Comment by thesmallstar** [comment]
@vorburger  should we not update the github PR template then? which says
image

**Comment by vorburger** [comment]
> LGTM. Just check the typo as the last line: `Multiple commits in singl PR make sense in certain cases (e.g. branch backports).`

Fixed, using Edit on GitHub UI. Which makes this PR itself now have 2 commits, and I'm too lazy to pull it locally and squash, which makes this an example of a PR that whichever committers merges this should squash... as this documents. :smile_cat: 

PS: @awasum et al just just [use GitHub Suggestions](https://www.google.com/search?channel=trow2&client=firefox-b-d&q=github+suggestion) to propose changes directly! (Instead of only "commenting".)

**Comment by thesmallstar** [comment]
> Fixed, using Edit on GitHub UI. Which makes this PR itself now have 2 commits, and I'm too lazy to pull it locally and squash, which makes this an example of a PR that whichever committers merges this should squash... as this documents. 😸

looks like we are about to have 3 commits as an example now :P

**Comment by vorburger** [comment]
> @vorburger should we not update the github PR template then? which says

great point / feedback, I didn't think of that! Thanks - done now.

> looks like we are about to have 3 commits as an example now :P

@thesmallstar yeah, you made me pull it after all (or perhaps I'm too stupid to know how to edit multiple files on GitHub's Web UI).

**Review by thesmallstar — APPROVED** [review]
LGTM!

**Review by Grandolf49 — APPROVED** [review]
LGTM

**Comment by thesmallstar** [comment]
@vorburger  this failed due to connection error (but I think you can merge this anyways)...

**Review by xurror — APPROVED** [review]
LGTM

**Review by percyashu — APPROVED** [review]
LGTM

File: .github/PULL_REQUEST_TEMPLATE.MD [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
 
 ## Checklist
+
 Please make sure these boxes are checked before submitting your pull request - thanks!
 
-- [ ] Commit message starts with the issue number from https://issues.apache.org/jira/projects/FINERACT/. Ex: FINERACT-646 Pockets API.
+- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests
 
-- [ ] Coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions have been followed.
+- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsability to get a proposed PR to pass the build, not primarily the project's maintainers.
```
Comment (ptuomola) [inline .github/PULL_REQUEST_TEMPLATE.MD]:
```suggestion
- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.
```

File: .github/PULL_REQUEST_TEMPLATE.MD [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
 
-- [ ] API documentation at fineract-provider/src/main/resources/static/api-docs/apiLive.htm has been updated with details of any API changes.
+- [ ] Create/update unit or integration tests for verifying the changes made.
 
-- [ ] Integration tests have been created/updated for verifying the changes made.
+- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions have been followed.
```
Comment (ptuomola) [inline .github/PULL_REQUEST_TEMPLATE.MD]:
```suggestion
- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.
```

File: .github/PULL_REQUEST_TEMPLATE.MD [hunk 3]
```diff
@@ -1,19 +1,24 @@
...
 
-- [ ] All Integrations tests are passing with the new commits.
+- [ ] Add required Swagger annotation and updated API documentation at fineract-provider/src/main/resources/static/api-docs/apiLive.htm with details of any API changes
```
Comment (ptuomola) [inline .github/PULL_REQUEST_TEMPLATE.MD]:
```suggestion
- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/api-docs/apiLive.htm with details of any API changes
```

File: .github/PULL_REQUEST_TEMPLATE.MD [hunk 4]
```diff
@@ -1,19 +1,24 @@
...
 
-- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the list.)
+- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailling list for guidance, if required.)
```
Comment (ptuomola) [inline .github/PULL_REQUEST_TEMPLATE.MD]:
```suggestion
- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)
```

**Pull request merged**